docker-for-aws/persistent-data-volumes.md

```diff
@@ -14,8 +14,8 @@ mode tasks and regular Docker containers can use a volume created with
 Cloudstor to mount a persistent data volume. In Docker for AWS, Cloudstor has
 two `backing` options:
 
-- `relocatable` data volumes are backed by EBS.
-- `shared` data volumes are backed by EFS.
+- `relocatable` data volumes which are backed by EBS.
```

Comment:
I don't think this is really needed. You took a full sentence and turned it into a fragment.